.env

```diff
@@ -2,6 +2,8 @@
 REACT_APP_SERVER_BASE_URL=https://www.anyway.co.il/
```

Comment:
Remove The word "Server" following naming (all URLs are for server):
`REACT_APP_API_BASE_URL` (instead of REACT_APP_SERVER_BASE_URL)
`REACT_APP_AUTH_BASE_URL` (instead of REACT_APP_AUTH_SERVER_BASE_URL)